tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -2997,6 +3000,15 @@ protected function parse_markup_fragment( $markup ) {
 			}
 		}
 
+		// Normalize other whitespace nodes.
+		$xpath = new DOMXPath( $dom );
+		foreach ( $xpath->query( '//text()' ) as $node ) {
+			/** @var DOMText $node */
+			if ( preg_match( '/^\s+$/', $node->nodeValue ) ) {
+				$node->nodeValue = ' ';
+			}
+		}
```

Comment:
see comment above: as a general test assertion I believe this will lead to false results because of the way `DOMDocument` fails to properly handle the inner contents of certain elements.

Reply:
Since tests here are limited to `script` elements, I think this is fine.